libs/perun/components/src/lib/attributes-list/attribute-value/attribute-value-string/attribute-value-string.component.scss

```
@@ -2,6 +2,19 @@
   cursor: pointer;
 }
 
+.hint-color {
+  color: darkorange;
+}
+
 .overflow-ellipsis {
   text-overflow: ellipsis;
 }
+//hide hint when field not in focus
+.mat-form-field:not(.mat-focused) .mat-hint {
```

I will have to change this component (fix the UI on global level) in my PR on migration to MDC, but at least this comment for this local scss file will be useful.
NOTE: I used `mat-hint` without the dot (`.`) on purpose, but if we want to use class definition, then we can use this one in Angular 15: `.mat-mdc-form-field-hint`.
```suggestion
// FIXME: After the migration to MDC and Angular 15 have to be replaced the line below
// .mat-mdc-form-field:not(.mat-focused) mat-hint {
.mat-form-field:not(.mat-focused) .mat-hint {
```